spec/workers/tag_subscription_reassignment_worker_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe TagSubscriptionReassignmentWorker, type: :worker do
+  before do
+    allow(described_class).to receive(:perform_async)
+    3.times{ create(:tag, name: 'tag') }
```

Comment:
Just an FYI, I don't know about the performance difference, but you could use `FactoryGirl`'s `create_list` like so:
```ruby
create_list(:tag, 3, name: 'tag')
```